Supporting duplicateses on multimap (#305)
infinispan/infinispan.github.io: _posts/2022-10-02-infinispan-14-supporting-duplicates-on-multimap.adoc

---
layout: blog
title: Infinispan 14 supporting duplicates on multimap
permalink: /blog/:year/:month/:day/infinispan-14-supporting-duplicates-on-multimap
date: '2022-10-2T00:00:00.000-00:00'
author: miador
tags: [ "multimap" ]
---

= Infinispan 14 supporting duplicates on multimap

Dear Infinispan community,

> with the Infinispan 14 development release 04 multimap can support duplicates.

Reviewer: ```suggestion
With the Infinispan 14 development release 04 multimap cache supports duplicates.
```
Just a suggestion @miador I would add one more sentence about the benefits that this brings. Why should I use it?

By default `supportsDuplicates` is set to false and can be configured during initialization.

> Here is an about how can we set multimap to support duplicate key-value pairs:

Reviewer: ```suggestion
The following is an example on how you can set multimap to support duplicate key-value pairs:
```
